src/main/java/org/commcare/resources/model/Resource.java

```diff
@@ -315,13 +332,15 @@ public Object getMetaData(String fieldName) {
                 return parent == null ? "" : parent;
             case META_INDEX_VERSION:
                 return version;
+            case META_INDEX_LAZY:
+                return lazy;
```

Comment:
I'm a little surprised to see this works with a boolean, btw. My intuition is that we should hard code an explicit string value here, and convert it to/from boolean. We've started making soft assumptions elsewhere that all metadata values are strings, and I'm worried that implicit conversions.